### ISSUE TITLE
[ENH]  populate no_compression_regexp with _something_ so it matches the documentation

### ISSUE THREAD

**Issue by speachy** [rdiff-backup/rdiff-backup#971]
The documentation claims "The default includes many common audiovisual and archive files, and may be found from the help." but the default in Globals.py is "None"  (And there's no other mention of this in the help)

Here's what I'd propose, off the top of my head:

Archive files: rar zip xz tgz gz bz2
Image files: webp png jpg jpeg nef crw dng jxl heif avci avif
Audio files: wma mp2 mp3 m4p m4a 
Video files: m4v mp4 mpg avi mkv asf wmv webm

**Comment by ikus060** [comment]
@speachy You look at the wrong place.

Here the default list: https://github.com/rdiff-backup/rdiff-backup/blob/44c34dc12aa11d08312b675dabf48e208ecb1d0d/src/rdiffbackup/actions/__init__.py#L35

Re-open this issue if you have questions related to this subject.

**Comment by speachy** [comment]
I stand corrected, my apologies.

However, the documentation specifically mentions that the default list may be found in the help text, but that doesn't seem to be the case?  (it's not in the man page, nor in the --help text for the --not-compressed-regexp option)

**Comment by ericzolf** [comment]
Fair enough,  the only issue is that I don't know how it'll look like,  the list being so long.

**Comment by ericzolf** [comment]
I had actually already fixed it in commit 8ae45f8f18ec604b8499b44bb18c4c8bd378c3d9 as I removed API 200 (for rdiff-backup 3.x), and here is how it looks like:

```console
$ rdiff-backup backup --help
usage: rdiff-backup backup [-h] [--create-full-path]
                           [--compression | --no-compression]
                           [--not-compressed-regexp REGEXP] [--exclude GLOB]
[...]
  --not-compressed-regexp REGEXP, --no-compression-regexp REGEXP
                        [sub] regexp to select files not being compressed
                        (default is '(?i).*\.(7z|aac|arj|asc|avi|bik|bz|bz2|de
                        b|docx|flac|flv|gif|gpg|gz|jp2|jpeg|jpg|jsonlz4|lharc|
                        lz4|lzh|lzma|lzo|m4a|m4v|mdf|mkv|mov|mozlz4|mp3|mp4|mp
                        eg|mpg|oga|ogg|ogm|ogv|opus|pgp|pk3|png|rar|rm|rpm|rz|
                        shn|tgz|tzst|vob|webm|webp|wma|wmv|xlsx|xz|z|zip|zoo|z
                        st)$')
```
